lib/modules/mod_tar.phtml

```diff
@@ -63,7 +63,7 @@
 				}
 			}
 			$result = $archive->addModify($filelist, "", $srcdir);
-			$archive->_Archive_Tar();
+			$archive->__destruct();
```

Comment:
isn't `unset($achive)` a cleaner way to do this ?

Reply:
agreed and fixed.